### PR TITLE
Add space for keys in calculation for rent exempt in `process_set_validator_info`

### PR DESCRIPTION
**Pull request by enriquefynn**
#### Problem
`process_set_validator_info` does not work as intended, thus the client's command `validator-info publish` does not work: leaves an account with space greater than the minimum rent exempt balance.

#### Summary of Changes
Add space for keys in calculation for rent exempt in `process_set_validator_info`.
The space required for allocating the `(ConfigKeys, ValidatorInfo)` tuple only considered space for `ValidatorInfo`
but `config_instruction::create_account` also requires space for `n` keys.

### CODE REVIEW

File: cli/src/validator_info.rs [hunk 1]
```diff
@@ -339,7 +339,7 @@ pub fn process_set_validator_info(
             let instructions = vec![config_instruction::store(
                 &info_pubkey,
                 false,
-                keys,
+                keys.clone(),
```
Comment (t-nelson) [inline cli/src/validator_info.rs:342]: nit: pretty sure these clones are unnecessary

**Review by t-nelson — APPROVED** [review]
lgtm! thanks for the fix!

File: cli/src/validator_info.rs (hunk 1 above)
Comment (enriquefynn) [reply cli/src/validator_info.rs]: Can clone it in the closure, so we save one clone call. Implemented it like this.

**Comment by CriesofCarrots** [comment]
Sorry about the delay here. On the docket for Monday!

**Pull request merged**